packages/app/src/app/components/Preview/DevTools/Console/Message/index.js

```diff
@@ -50,6 +59,24 @@ function getMessage(message) {
   }
 
   if (message.arguments.every(argument => typeof argument === 'string')) {
+    if (message.logType === 'error') {
+      let otherErrorLines;
+      const msgLine = message.arguments.join(' ');
+      const firstLine = splitMessage(msgLine);
+      const msgArray = msgLine.split('\n');
+      if (msgArray.length > 1) {
+        otherErrorLines = msgArray.slice(1);
+      }
+      return (
+        <InnerItem>
+          <details>
+            <summary>{firstLine}</summary>
+            {otherErrorLines}
```

Comment:
I think this should become `otherErrorLines.join('\n\r')` to make it work again 😄